src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -47,7 +48,27 @@
 }
 
 
-def histogram(polygons):
+def histogramPoll(jobId):
+    host = settings.GEOP['host']
+    port = settings.GEOP['port']
+    path = '/jobs/' + jobId
+    conn = httplib.HTTPConnection("%s:%s" % (host, port))
```

Comment:
Consider pulling out the shared network code of `histogram` and `histogramPoll` into a shared helper function.

Also consider using `requests` (an external [library](https://github.com/kennethreitz/requests)), which is less verbose than and generally easier to use than `httplib`.